packages/scene-composer/src/hooks/useSceneModal.tsx

```diff
@@ -22,6 +28,14 @@ const useSceneModal = (): React.JSX.Element | null => {
     return <MessageModal />;
   }
 
+  if (deleteConfirmationModalVisible) {
+    if (deleteConfirmationModalVisibleParams?.type === 'deleteNode') {
```

Comment:
We should create an enum for this - `DeleteConfirmationModalType` with values `deleteNode`, `deleteComponent`. Use this enum here and in the function calls, instead of using hard-coded strings everywhere.

Reply:
the type of the `type` field is defined as string literal unions, so it will get all the type check benefits and more light weighted than enum https://contra.com/p/W3ol7m3o-enums-vs-string-literal-unions-in-type-script

Reply:
Ok.. Then we should at least move these hard-coded strings as `const` at a common place and use it everywhere.

Reply:
This can also be included in next change as well.

Reply:
we can't use const in type definition since it's value